DSharpPlus/Entities/Guild/DiscordGuild.cs

```diff
@@ -503,7 +504,43 @@ internal DiscordGuild()
         }
 
         #region Guild Methods
+        
+        /// <summary>
+        /// Gets guild's icon URL, in requested format and size.
+        /// </summary>
+        /// <param name="imageFormat">The image format of the icon to get.</param>
+        /// <param name="imageSize">The maximum size of the icon. Must be a power of two, minimum 16, maximum 4096.</param>
+        /// <returns>The URL of the guild's icon.</returns>
+        public string GetIconUrl(ImageFormat imageFormat, ushort imageSize = 1024)
+        {
+
```

Comment:
According to current implementation of `IconUrl, `IconHash` property can be null or whitespaced, and in that case `IconUrl` returns `null`. Check for it should be added to this method too.